src/Microsoft.DotNet.Build.Tasks/Targets/tests.targets

```diff
@@ -19,20 +19,51 @@
     <UnsupportedPlatformsItems Include="$(UnsupportedPlatforms)"/>
   </ItemGroup>
 
+  <PropertyGroup>
+    <TestWithCore Condition="'$(TestWithCore)' == ''">true</TestWithCore>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TestWithCore)' == 'true'">
+    <TestRuntimeProjectJson Condition="'$(TestRuntimeProjectJson)' == ''">$(MSBuildThisFileDirectory)test-runtime\project.json</TestRuntimeProjectJson>
+    <TestRuntimeProjectLockJson Condition="'$(TestRuntimeProjectLockJson)' == ''">$(MSBuildThisFileDirectory)test-runtime\project.lock.json</TestRuntimeProjectLockJson>
+    <TestHost>corerun.exe</TestHost>
+    <XunitExecutable Condition="'$(XunitExecutable)' == ''">xunit.console.netcore.exe</XunitExecutable>
+    <DebugTestFrameworkFolder>dnxcore50</DebugTestFrameworkFolder>
+    <DebugEngines>{2E36F1D4-B23C-435D-AB41-18E608940038}</DebugEngines>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TestWithCore)' != 'true'">
+    <XunitExecutable Condition="'$(XunitExecutable)' == ''">xunit.console.exe</XunitExecutable>
+    <DebugTestFrameworkFolder>dnx45</DebugTestFrameworkFolder>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(TestWithCore)' == 'true'">
+    <TestTargetFramework Include="DNXCore,Version=v5.0">
+      <Folder>dnxcore50</Folder>
+    </TestTargetFramework>
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TestWithCore)' != 'true'">
+    <TestTargetFramework Include="DNX,Version=v4.5">
```

Comment:
Yes, don't use DNX here.  The only reason we are using DNXCore above is for asset selection (we want the libraries that are coincidentally set up for the same OS/arch pivots as DNX).  Eventually we should probably switch to netcore, or potentially "dotnet" for the coreclr based test run if we are able to make the runtime/facade packages work with that.